revolv/static/dashboard.scss

```diff
@@ -401,6 +401,103 @@ $sidebar-icon-gray: #999;
     }
 }
 
+$light-gray: #c6c6c6;
+$dark-gray: #555555;
+
+.dashboard-data-section-my-impact {
+    height: 100%;
+
+    .category-preferences-container {
+        height: 100%;
+        padding: 0;
+
+        .gray-bar-container {
+            height: 100%;
+            padding: 0;
+            
+            .gray-bar {
+                position: absolute;
+                height: 84%;
+                top: 8%;
+                background-color: $light-gray;
+                width: 1px;
+            }
+        }
+
+        .choice-container {
+            padding: 0;
+
+            .choice-header {
+                font-weight: 800;
+                color: $dark-gray;
+                text-align: center;
+                margin: 30px 15px;
+                font-size: 1.1rem;
+            }
+
+            .category-option-container {
+                width: 75%;
+                height: 40px;
+                border: 1px solid $light-gray;
+                margin: 5px auto 0 auto;
+                transition: all 0.3s;
```

Comment:
It might if the page resizes right? Or if we had a really long category name which went on two lines at small sizes, the width would animate? Also isn't animating everything going to make it slower? I thought I heard that somewhere